pkg/app/pipedv1/plugin/kubernetes/deployment/plugin.go

```diff
@@ -348,8 +348,8 @@ func (p *Plugin) DetermineVersions(context.Context, *sdk.ConfigNone, *sdk.Client
 }
 
 // FIXME
-func (p *Plugin) DetermineStrategy(context.Context, *sdk.ConfigNone, *sdk.Client, sdk.TODO) (sdk.TODO, error) {
-	return sdk.TODO{}, nil
+func (p *Plugin) DetermineStrategy(context.Context, *sdk.ConfigNone, *sdk.Client, *sdk.DetermineStrategyInput) (*sdk.DetermineStrategyResponse, error) {
```

Comment:
Remove the comment FIXME on as function description :eyes:

Reply:
I haven't implemented this method yet; just changed the signature.

Reply:
Got it, thanks 👍